provision/roles/inventory_gen/tasks/main.yml

```diff
@@ -33,9 +33,9 @@
     msg: "{{ inventory_layout }}"
   when: state == "present" and inventory_layout is defined
 
-# include inventory generation files here : can be overridden by  extravar inventory_layout_file currently defaulted to duffy_3node_inventory_layout.yml
+# include inventory generation files here : can be overridden by  extravar inventory_layout_file currently defaulted to openshift-3node-cluster.yml
 - name: "include the layout file for inventory generation"
-  include_vars: "{{ inventory_layout_file | default( inventory_layouts_path+'/'+'duffy_3node_inventory_layout.yml') }}"
+  include_vars: "{{ inventory_layout_file | default( inventory_layouts_path+'/'+'openshift-3node-cluster.yml') }}"
```

Comment:
I'm confused by this change. Why would we hardcode the inventory_layout_file? It's either set at runtime with --extra-vars, or provided in a config file (like linchpin_config.yml), right?